assets/js/amp-editor-blocks.js

```diff
@@ -113,6 +113,7 @@ const ampEditorBlocks = ( function() {
 		wp.hooks.addFilter( 'blocks.getSaveElement', 'ampEditorBlocks/filterSave', component.filterBlocksSave );
 		wp.hooks.addFilter( 'editor.BlockEdit', 'ampEditorBlocks/filterEdit', component.filterBlocksEdit );
 		wp.hooks.addFilter( 'blocks.getSaveContent.extraProps', 'ampEditorBlocks/addExtraAttributes', component.addAMPExtraProps );
+		wp.domReady( component.maybeUnregisterBlocks );
```

Comment:
Does calling this immediately (and not waiting for `domReady`) help with the error?

Reply:
Great idea, https://github.com/ampproject/amp-wp/pull/1916/commits/01bafa706c3e22c180326ba68c4f603f3d13a6a4 calls it right away, without `wp.domReady`. 

Now, the warning is as expected when converting to Classic mode:
<img width="1180" alt="warning-as-expected" src="https://user-images.githubusercontent.com/4063887/54051345-39686e80-41a7-11e9-99dc-3cd237133759.png">

Reply:
So there are no more outstanding issues as far as you're concerned, and it is ready for final review?

Reply:
Yes, sorry for the delay in replying.